### ISSUE TITLE
Turbo not loaded when following 7.0.1 tutorial

### ISSUE THREAD

**Issue by Rik-de-Kort** [rails/rails#44229]
### Steps to reproduce
Following [Getting Started with Rails](https://guides.rubyonrails.org/getting_started.html#deleting-an-article) in the selection on "deleting an article". Clicking the destroy link does nothing, just produce a GET-request for the article itself. Look at the generated HTML, turbo is never loaded. Adding in `<%= turbo_include_tags %>` in `app/views/articles/show.html.erb` fixes it.

### Expected behavior
Clicking 'Destroy' on the article page removes the article by sending a DELETE-request, with a confim dialog beforehand.

### Actual behavior
Turbo is never loaded, so clicking 'Destroy' simply sends a GET-request to the article.

### System configuration
**Rails version**: 7.0.1

**Ruby version**: ruby 3.0.3p157 (2021-11-24 revision 3fb7d2cadc) [x86_64-linux]

On Arch, I also had some issues with #44143

Is there anything I can do to make sure Turbo is being included without having to remember to type turbo_include_tags all the time?

Also, stylesheets appear not to be loaded either, as my app doesn't correspond to the screenshot in the guide.

**Comment by rohandaxini** [comment]
Hi @Rik-de-Kort 
From the [Getting Started with Rails](https://guides.rubyonrails.org/getting_started.html#deleting-an-article), deleting an article using `turbo_method: :delete` is working perfectly fine for me. When I look at the generated HTML, turbo is properly loaded for me. I can see that the `imports` section for me has proper turbo-rails loaded.

```
"@hotwired/turbo-rails": "/assets/turbo.min-96cbf52c71021ba210235aaeec4720012d2c1df7d2dab3770cfa49eea3bb09da.js",
```

I am on Rails `7.0.1` and Ruby `3.0.0`

I am 🤔 of you are facing this issue > https://github.com/hotwired/turbo-rails/issues/66
Ideally, you should not as you are on Rails 7.0.1 

If I find more details then ill add another comment for you soon here.

**Comment by Rik-de-Kort** [comment]
Recreating the app from scratch and reinstalling the rails gem seemed to help. However, I'm still getting the rdoc error if I don't add it to my gemfile.
It did prompt me for my password, so maybe there's some global installation thing that went awry?

**Comment by mydoomfr** [comment]
I was facing the same issue on a project initialized with the first release of rails 7.0.0. After upgrading to 7.0.1 on the same project, the issue still occurs. The workaround of adding `<%= turbo_include_tags %>` fix the issue.

Like @Rik-de-Kort, I've ended up recreating a new project from scratch and the issue is now solved.

```
ruby --version
ruby 3.1.0p0 (2021-12-25 revision fb4df44d16) [arm64-darwin21]
```

**Comment by WeebNetsu** [comment]
I think this issue should be reopened until the workaround/fix is added on the ROR guide, or is at least mentioned...

**Comment by skipkayhil** [comment]
> I think this issue should be reopened until the workaround/fix is added on the ROR guide, or is at least mentioned

I don't think the `<%= turbo_include_tags %>` solution should be recommended because it shouldn't be needed at all. Generally, every issue I've seen where people are having issues with Turbo not loading is due to some error while running `rails new`. If `rails new` runs without errors, `import "@hotwired/turbo-rails"` is added to `application.js` and Turbo will be loaded.

If you (or anyone) can reproduce an error while running `rails new`, you should absolutely open an issue detailing the steps and the full log of running `rails new`. Otherwise, I don't think there is anything actionable here.

**Comment by vladsf** [comment]
I have the same issue with Rails 7.0.4.2.

**Comment by vladsf** [comment]
Also `rails new` alwaus ends like this
```
Your user account isn't allowed to install to the system RubyGems.
  You can cancel this installation and run:

      bundle config set --local path 'vendor/bundle'
      bundle install

  to install the gems into ./vendor/bundle/, or you can enter your password
  and install the bundled gems to RubyGems using sudo.
```

**Comment by sunwooz** [comment]
I'm on Rails 7.0.5 and `<%= turbo_include_tags %>` finally fixed the problem for me.

**Comment by yisuren** [comment]
same issue with the start up guide, when click `destroy` link

ruby -v
ruby 3.2.2

rails --version
Rails 7.1.1

`<%= turbo_include_tags %>` finally fixed the problem

**Comment by luco17** [comment]
Same issue, couldn't delete the article until I added `<%= turbo_include_tags %>`

```
ruby --version
ruby 3.3.1 (2024-04-23 revision c56cd86388) [arm64-darwin23]
```

```
rails --version
Rails 7.1.3.2
```

I created the project through RubyMine.

**Comment by nathantinius** [comment]
Followed the [Getting Started with Rails ](https://guides.rubyonrails.org/getting_started.html#deleting-an-article) guide after installing ruby and rails on a brand new machine via Homebrew.  The same issue persists for me as well. 

- I use the workaround mentioned above: <%= turbo_include_tags %> in show.html.erb continues to work in version 7.2.1.

`ruby 3.3.4 (2024-07-09 revision be1089c8ec) [arm64-darwin23]`
`Rails 7.2.1`

**Comment by test0n3** [comment]
Greetings, have not tried the bundle configuration mentioned above. but `<%= turbo_include_tags %>` included in *application.html.erb* seems to work.

`Ruby: 3.3.5`
`Rails: 7.2.1`
Playing with Docker.

**Comment by UmairAhmedImran** [comment]
@test0n3 including `<%= turbo_include_tags %>` will work but later on it can cause more problems. So I will recommend to have rails new <application-name> to run properly.  Check  the image below on left side I created the application with --skip-bundle and it does not include app/javascript. But when I create a new app without any tags it created app/javascript and the code run smoothly without using any `<%= turbo_include_tags %>`
![github](https://github.com/user-attachments/assets/1e1341f2-5476-4c86-96b7-b3f3e5e8c52a)

**Comment by test0n3** [comment]
@UmairAhmedImran checking how to handle the configuration I used with this rails installation: `--skip-bundle`. Don't know yet if it's reversible at all. But, why that argument stopped the generation of the directories and files? 

Found this: [StackOverflow - Ruby on Rails - javascript(s) folder and application.js missing](https://stackoverflow.com/questions/72234526/ruby-on-rails-javascripts-folder-and-application-js-missing) and helped a little.

```
bin/rails importmap:install
bin/rails turbo:install # got this one from bin/rails new --help
```
for now, it seems to work.
